DNN Platform/Tests/DotNetNuke.Tests.Web/InternalServices/FileUploadControllerTests.cs

```diff
@@ -54,6 +54,13 @@ public void SetUp()
             this.SetupSynchronizationContext();
         }
 
+        public void TearDown()
```

Comment:
You'll need the attribute, too.
```suggestion
        [TearDown]
        public void TearDown()
```

Reply:
oops I missed it! Fixed now.